query_processor/validators.py

```
@@ -0,0 +1,266 @@
+        url: URL to validate
+        
+    Returns:
+        bool: True if valid YouTube URL
+    """
+    if not url or not url.strip():
+        return False
+    
+    try:
+        validate_youtube_url(url.strip())
+        return True
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)